package.json

```diff
@@ -24,11 +24,17 @@
     "cli-color": "^2.0.2",
     "fs-extra": "^10.1.0",
     "mocha": "^8.3.2",
+    "node-gyp": "8.4.1",
```

Comment:
`^`?

Reply:
Although the number of times I've seen non-semver-compliant changes in npm packages is decidedly nonzero. It's safer to pin it, probably.

Reply:
Sqlite3 for example already has node-gyp 8.4.1 as dependency. We can remove the dependency, if a users global installation should be used for better-sqlite3. But when it comes down to run prebuild, the overwrite would still be required, as prebuild would always use its internal node-gyp 6.1.0, which is not able to build for electron v20.